packages/graph-explorer/src/connector/LoggerConnector.ts

```diff
@@ -1,14 +1,21 @@
+import { logger } from "../utils";
+
 export type LogLevel = "error" | "warn" | "info" | "debug" | "trace";
 
-type Options = { enable?: boolean };
-export default class LoggerConnector {
+interface ILoggerConnector {
+  error(message: unknown): void;
+  warn(message: unknown): void;
+  info(message: unknown): void;
+  debug(message: unknown): void;
+  trace(message: unknown): void;
+}
+
+export class LoggerConnector implements ILoggerConnector {
```

Comment:
Class name is a bit confusing. May be better to use something to help understand the difference with `ClientLoggerConnector`?
like `ProxyLoggerConnector`?

Reply:
Fair point. Will rename.